lib/fastlane/plugin/firebase_app_distribution/helper/firebase_app_distribution_helper.rb

```diff
@@ -28,10 +28,10 @@ def parse_plist(path)
         CFPropertyList.native_types(CFPropertyList::List.new(file: path).value)
       end
 
-      def get_ios_app_id_from_archive(path)
-        app_path = parse_plist("#{path}/Info.plist")["ApplicationProperties"]["ApplicationPath"]
-        UI.shell_error!("can't extract application path from Info.plist at #{path}") if app_path.empty?
-        identifier = parse_plist("#{path}/Products/#{app_path}/GoogleService-Info.plist")["GOOGLE_APP_ID"]
+      def get_ios_app_id_from_archive_plist(archive_path, plist_path)
```

Comment:
I'm not sure why the tests didn't run against this PR (maybe because it's from before we had CI hooked up) but could you please update the spec here to include the new `plist_path` argument? https://github.com/fastlane/fastlane-plugin-firebase_app_distribution/blob/master/spec/firebase_app_distribution_helper_spec.rb#L75

You can run the specs by running `rake` from the plugin project root.

Reply:
I updated the spec and now launching `bundle exec rake -t` passes the tests, however I have no experience in RSpec so please correct me if I'm mistaken

Reply:
Looks great, thanks!